LabDrivers/utils.py

```diff
@@ -743,11 +744,35 @@ def close(self):
         #self.reset()
         pass
 
-    def control_ren(self, level):
+    def control_ren(self, level, addr = None):
+        if level is None:
+            return
+        ASSERT_REN = [1,3]
```

Comment:
why don't you assign global variables in the higher scope of the module? (i.e., why inside a function?)

Reply:
I did not assign it as a global variable because it is only used in this function, and will never be subject to change.